setup.py

```diff
@@ -72,22 +72,23 @@
             # XXX: Intersphinx URLs are broken when using non-default inventory
             # (like in Django's case) with Sphinx 1.4.3+
             # Bug report: https://github.com/sphinx-doc/sphinx/issues/2789
-            'Sphinx==1.4.2',
+            'sphinx==1.4.2',
```

Comment:
I would leave `Sphinx` with a capital `S` since browsing to https://pypi.python.org/pypi/sphinx redirects you to https://pypi.python.org/pypi/Sphinx, which indicates that `Sphinx` is the correct name to use.